examples/jira_mysql_nativessl_install.pp

```diff
@@ -0,0 +1,40 @@
+node default {
+
+  class { '::mysql::server':
+    root_password    => 'strongpassword',
+  } ->
+
+  mysql::db { 'jira':
+    user     => 'jiraadm',
+    password => 'mypassword',
+    host     => 'localhost',
+    grant    => ['ALL'],
+  } ->
+
+  exec { 'tmpkey':
```

Comment:
Does it exist on CentOS and RHEL? I thought it was just in Debian based releases.

Possibly I should remove this exec completely from the example?